.github/workflows/block-build.yml

```diff
@@ -34,17 +34,20 @@ jobs:
         uses: actions/cache@v2
         id: cache
         with:
-          key: Block-${{ runner.os }}-${{ matrix.node-version }}-${{ hashFiles('**/package-lock.json') }}
+          key: cache-${{ runner.os }}-${{ matrix.node-version }}-${{ hashFiles('**/package-lock.json') }}
```

Comment:
I believe the latest version of actions/setup-node does this automatically for you. I haven't tried that yet, but looks promising.

https://github.com/actions/setup-node#caching-packages-dependencies

Reply:
oh, nice, thanks! that does look a lot simpler. let me give it a try.